clang/lib/Driver/ToolChains/Gnu.cpp

```diff
@@ -618,6 +618,7 @@ void tools::gnutools::Linker::ConstructJob(Compilation &C, const JobAction &JA,
         CmdArgs.push_back("--push-state");
```

Comment:
Is Issue #41880 and D118021 still relevant? If not, the description needs to describe why the change is now safe.